packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -96,20 +96,20 @@ public class ProjectController {
 
 	static final String WELCOME_MESSAGE =
 			"""
-		<div>
-			<h2>Hey there!</h2>
-			<p>This is your project overview page. Use this space however you like. Not sure where to start? Here are some things you can try:</p>
-			<br>
-				<ul>
-					<li><strong>Upload stuff:</strong> Upload documents, models, code or datasets with the green button in the bottom left corner.</li>
-					<li><strong>Explore and add:</strong> Use the project selector in the top nav to switch to the Explorer where you can find documents, models and datasets that you can add to your project.</li>
-					<li><strong>Build a model:</strong> Create a model that fits just what you need.</li>
-					<li><strong>Create a workflow:</strong> Connect resources with operators so you can focus on the science and not the plumbing.</li>
-				</ul>
-			<br>
-			<p>Feel free to erase this text and make it your own.</p>
-		</div>
-		""";
+			<div>
```

Comment:
How is this block's formatting changed? i would have thought the linter would have stopped that?